neuraxle/base.py

```diff
@@ -379,29 +461,45 @@ def inverse_transform(self, processed_outputs):
         """
         return processed_outputs
 
-    def inverse_transform_one(self, processed_output):
-        """
-        Do nothing - return the same data.
-
-        :param processed_output: the data to process
-        :return: the ``data_output``, unchanged.
-        """
-        return processed_output
-
 
 class TruncableSteps(BaseStep, ABC):
 
     def __init__(
             self,
             steps_as_tuple: NamedTupleList,
             hyperparams: HyperparameterSamples = dict(),
-            hyperparams_space: HyperparameterSpace = dict()
+            hyperparams_space: HyperparameterSpace = dict(),
+            hasher: BaseHasher = None
     ):
-        super().__init__(hyperparams, hyperparams_space)
+        super().__init__(hyperparams=hyperparams, hyperparams_space=hyperparams_space, hasher=hasher)
         self.steps_as_tuple: NamedTupleList = self.patch_missing_names(steps_as_tuple)
         self._refresh_steps()
+        self.is_initialized = False
+
         assert isinstance(self, BaseStep), "Classes that inherit from TruncableMixin must also inherit from BaseStep."
 
+    def setup(self, step_path: str = None, setup_arguments: dict = None):
+        if self.is_initialized:
+            return
+
+        if step_path is None:
+            step_path = self.name
+
+        if setup_arguments is None:
+            setup_arguments = {}
```

Comment:
I'd perhaps would like to do here what's done in `TruncableSteps.set_hyperparams` and rename keys of the dict without underscores, etc. 

However, what I suggest would be over-engineering for now as there would be no use case to this yet, so your code is fine as it is currently.

Reply:
won't fix in this pr